include/roboptim/core/util.hxx

```diff
@@ -170,7 +170,7 @@ namespace roboptim
     ROBOPTIM_ASSERT (startRow + b.rows () <= m.rows ());
     ROBOPTIM_ASSERT (startCol + b.cols () <= m.cols ());
 # if __cplusplus < 201103L || !defined (__GXX_EXPERIMENTAL_CXX0X__)
-    ROBOPTIM_ASSERT (int (M::IsRowMajor) == int (B::IsRowMajor));
+    ROBOPTIM_ASSERT ( ROBOPTIM_UNUSED int (M::IsRowMajor) == int (B::IsRowMajor));
```

Comment:
I noticed that `ROBOPTIM_UNUSED` is actually [wrong](https://github.com/roboptim/roboptim-core/blob/11e0e1954c361a8bad6827fb403b8d62e4a781bc/include/roboptim/core/portability.hh#L28) for Win32. That may be part of the problem.